src/metabase/analytics/stats.clj

```diff
@@ -203,8 +204,11 @@
   "Get metrics based on dashboards
   TODO characterize by # of revisions, and created by an admin"
   []
-  (let [dashboards (t2/select [Dashboard :creator_id :public_uuid :parameters :enable_embedding :embedding_params])
-        dashcards  (t2/select [DashboardCard :card_id :dashboard_id])]
+  (let [dashboards (t2/select [Dashboard :creator_id :public_uuid :parameters :enable_embedding :embedding_params] :creator_id [:not= config/internal-mb-user-id])
+        dashcards  (t2/query {:select :dc.*
+                              :from [[(t2/table-name DashboardCard) :dc]]
+                              :join [[(t2/table-name Dashboard) :d] [:= :d.id :dc.dashboard_id]]
+                              :where [:not [:= :d.creator_id config/internal-mb-user-id]]})]
```

Comment:
It's probably more efficient to flip this around and skip the join by doing something like `[:not [:in :dc.dashboard_id excluded-dashboard-ides]]`

Reply:
Does it matter? This query gets every dashcard in the app DB anyway

Reply:
My intuition for how databases work, and the related costs, is very rotted, so I may be way off.

I suspect that a table scan with a join is a lot more expensive than a plain table scan, I'd expect it to build a temporary table or do sub-selects.

I don't think the query analyzer has access to statistics that would suggest doing this inversion internally, and it's not any more complicated. Not gonna block on it, this namespace is ripe for broader optimizations.